fastcov.py

```diff
@@ -249,6 +250,30 @@ def removeFiles(files):
     for file in files:
         os.remove(file)
 
+def processPrefix(path, prefix, prefix_strip):
```

Comment:
Do you mind adding a few unit tests for this function in:
https://github.com/RPGillespie6/fastcov/tree/master/test/unit

You can use https://github.com/RPGillespie6/fastcov/blob/master/test/unit/test_utils.py as a template.

If too much of a hassle, I can do it. Basically I would just add something like `test_gcov_prefix_options.py` with some tests and it should get automatically picked up in CI/CD pipeline.

Reply:
Unit tests added. I believe this covers all requested changes.

Reply:
Test cases look good, thanks for doing that